.circleci/ds005_outputs.txt

```diff
@@ -6,60 +6,64 @@ fmriprep/logs/CITATION.md
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-fsaverage5.R.func.gii
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-MNI152NLin2009cAsym_brainmask.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-MNI152NLin2009cAsym_preproc.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-MNI152NLin2009cAsym_variant-smoothAROMAnonaggr_preproc.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-T1w_brainmask.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-T1w_label-aparcaseg_roi.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-T1w_label-aseg_roi.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-T1w_preproc.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_AROMAnoiseICs.csv
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_confounds.tsv
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_MELODICmix.tsv
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-cifti_variant-space1_preproc.dtseries.json
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-cifti_variant-space1_preproc.dtseries.nii
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-fsaverage5.L.func.gii
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-fsaverage5.R.func.gii
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-MNI152NLin2009cAsym_brainmask.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-MNI152NLin2009cAsym_preproc.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-MNI152NLin2009cAsym_variant-smoothAROMAnonaggr_preproc.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-T1w_brainmask.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-T1w_label-aparcaseg_roi.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-T1w_label-aseg_roi.nii.gz
-fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-02_bold_space-T1w_preproc.nii.gz
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_AROMAnoiseICs.csv
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_confounds.tsv
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_MELODICmix.tsv
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_space-cifti_variant-space1_preproc.dtseries.json
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_space-cifti_variant-space1_preproc.dtseries.nii
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_space-fsaverage5.L.func.gii
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_space-fsaverage5.R.func.gii
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_space-MNI152NLin2009cAsym_brainmask.nii.gz
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_space-MNI152NLin2009cAsym_desc-3dref_bold.nii.gz
```

Comment:
I don't think this qualifies as a `bold.nii.gz`, as it's not a 4D time series. We didn't try to get a `ref` suffix (or perhaps `boldref`, since there is no longer a `bold` entry) into RC1, but we can still use it as long as there isn't a more appropriate suffix in the spec.

Pinging BIDS-meister @chrisfilo for a second opinion, though.